db/migrate/20201109114802_create_messages.rb

```diff
@@ -0,0 +1,10 @@
+class CreateMessages < ActiveRecord::Migration[6.0]
+  def change
+    create_table :messages do |t|
+      t.string :name
+      t.integer :age
```

Comment:
カラム名や型もmessageの場合に変更しなければなりません。

```suggestion
      t.string :title
      t.string :content
```

に変更した上で，以下のコマンドを実行して反映させて下さい。
（Railsサーバーを起動中の場合は先に停止させて下さい）

```
rails db:migrate:reset
```